src/Context.php

```diff
@@ -72,7 +72,13 @@ public function initialize(string $format, VisitorInterface $visitor, GraphNavig
         $this->metadataStack = new \SplStack();
 
         if (isset($this->attributes['groups'])) {
-            $this->addExclusionStrategy(new GroupsExclusionStrategy($this->attributes['groups']));
+            $strategy = new GroupsExclusionStrategy($this->attributes['groups']);
```

Comment:
why not 

```php

$g = new GroupsExclusionStrategy(['a', 'b'], true);
$context->addExclusionStrategy($g);
```
?

In that way we do not need to add the `enableInheritGroups` method in the context. Ideally the context should know the less possible about the various exclusion strategies.

Reply:
Ok, i see, but i can't add 2 GroupsExclusionStrategy, one with inheritedGroups = false and an another one with inheritedGroups = true.

How can i configure this option with JMSerializerBundle for example ?

Thanks for you help